src/RoleAdmin.php

```diff
@@ -35,8 +35,7 @@ public function setModel(Model $role, array $fields = null): void
             Header::addTo($v, [$role->getTitle() . ' Permissions']);
 
             $crud = Crud::addTo($v);
-            //$crud->setModel($role->ref('AccessRules')); // this way it adds wrong table alias in field condition - ATK bug (withTitle + table_alias)
-            $crud->setModel((new AccessRule($role->persistence))->addCondition('role_id', $id));
+            $crud->setModel($role->ref('AccessRules'));
```

Comment:
fixed in https://github.com/atk4/data/pull/938/commits/e77d3bc7d0d449f6c9cd603012501a087d31b36d